embedded_gems/pathogen/lib/pathogen/styles/form_styles.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Pathogen
+  module Styles
+    # Shared form styling constants and utility methods
+    #
+    # This module provides shared styling constants and utility methods
+    # that can be used across different form components and helpers.
+    module FormStyles
+      # Common styling classes for checkboxes
+      CHECKBOX_CLASSES = [
+        'size-6', # 24px - better accessibility
+        'bg-slate-100',
```

Comment:
suggestion (blocking): Can you change this to `bg-white` which helps the checkbox stand out more in light mode.

Reply:
Great idea, updated in bf3b67d0940f93f4365e688dafaca806c6cdb4f4